modin/pandas/test/utils.py

```diff
@@ -679,15 +679,15 @@ def modin_df_almost_equals_pandas(modin_df, pandas_df):
     if hasattr(pandas_df, "select_dtypes"):
         pandas_df = pandas_df.select_dtypes(exclude=["category"])
 
-    difference = modin_df - pandas_df
-    diff_max = difference.max()
-    if isinstance(diff_max, pandas.Series):
-        diff_max = diff_max.max()
-    assert (
-        modin_df.equals(pandas_df)
-        or diff_max < 0.0001
-        or (all(modin_df.isna().all()) and all(pandas_df.isna().all()))
-    )
+    if modin_df.equals(pandas_df) or (
+        all(modin_df.isna().all()) and all(pandas_df.isna().all())
+    ):
+        return
+
+    diff = (modin_df - pandas_df).abs()
+    diff /= modin_df.where(modin_df > pandas_df, pandas_df)
```

Comment:
?
```suggestion
    diff /= pandas_df
```

Reply:
To get the difference in %, it must be divided by the largest value.

Reply:
I thought we have to divide to a reference value (which is `pandas_df` in our case), not by the largest? https://en.wikipedia.org/wiki/Relative_change_and_difference#Definition

Reply:
This is a percentage difference between 2 numbers, but not a difference with a reference value.  It would be more properly to divide by (pandas_df + modin_df)/2, but selecting the largest value is also ok for this purpose, as I think.
Anyway, I do not insist. Dividing by pandas_df is also ok for this use case.

Reply:
Replaced with diff /= pandas_df.